src/mon/PGMap.cc

```diff
@@ -3348,9 +3348,14 @@ void PGMap::get_health_checks(
       // application metadata is not encoded until luminous is minimum
       // required release
       if (pool.application_metadata.empty() && !pool.is_tier()) {
-        stringstream ss;
-        ss << "application not enabled on pool '" << pool_name << "'";
-        detail.push_back(ss.str());
+        const auto now = ceph::real_clock::to_double(ceph::real_clock::now());
+        auto create_time = pool.get_create_time();
+        if ((now - create_time) >
```

Comment:
I might have confused you with my "might want to make the use of `CLOCK_REALTIME` explicit" comment.  Involving double isn't necessary and `const` is pretty bogus here:
```suggestion
        utime_t now(ceph::real_clock::now());
        if ((now - pool.get_create_time()) >
```

Reply:
It's okay. I am going to fix this.  😃